src/frontend/components/Felleskomponenter/Datovelger/Datovelger.test.tsx

```diff
@@ -27,8 +27,6 @@ class ResizeObserver {
     }
 }
 
-const IntlProviderMedLocale = ({ children }) => <IntlProvider locale="nb">{children}</IntlProvider>;
```

Comment:
`IntlProvider` er den gamle måten å hente ut tekster på, fra JSON-filene som @ole-kristian-rudjord jobber med å ta over til Sanity. Fjerner den herfra da den ikke trengs lenger for denne komponenten. 🎉